cumulusci/salesforce_api/org_schema.py

```diff
@@ -348,3 +499,31 @@ def deep_describe(
             for response in responses
         )
         yield from responses
+
+
+def ignore_based_on_properties(obj: dict, filters: T.Sequence[Filters]):
+    if all(obj.get(filter.name, True) for filter in filters):
+        return False
+    else:
+        return True
+
+
+NOT_COUNTABLE = (
+    "ContentDocumentLink",  # ContentDocumentLink requires a filter by a single Id on ContentDocumentId or LinkedEntityId
+    "ContentFolder%",  # Implementation restriction: ContentFolderItem requires a filter by Id or ParentContentFolderId
+    "IdeaComment",  # you must filter using the following syntax: CommunityId = [single ID],
+    "Vote",  # you must filter using the following syntax: ParentId = [single ID],
+    "RecordActionHistory",  # Gack: 1133111327-118855 (1126216936)
+)
+
+
+NOT_EXTRACTABLE = NOT_COUNTABLE + (
```

Comment:
I think this list is risky. It's valid to name a custom object `User_Permission__c`, for example, and if I read right this would result in it being excluded.

Share records _are_ migrateable but we will not handle them well for other reasons (polymorphic relations). 

`UserRecordAccess` and similar are more like a view than a real object. Can we exclude them based on either explicit names or some other facet of their schema to avoid collateral damage?

Reply:
Yes, you're right. This was hacky stuff I did while I was exploring and forgot to clean up later. I will blow these up to explicit lists.

Reply:
@davidmreed Your comment was addressed in 99e1f3d